### PR TITLE
Rating, slider

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
- Added rating component and docs
- added slider component and docs

### CODE REVIEW

File: components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+.simple-slider{
+    margin: 0 2px;
+}
+
+.styled-slider{
+    appearance: none;
+}
+
+.styled-slider::-webkit-slider-runnable-track{
+    background-color: var(--primary-color);
```
Comment (arulk8) [inline components/Slider/slider.css:17]: what this will do?

**Review by arulk8 — COMMENTED** [review]
Good to go.

File: components/Rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                style="width: 100%; height: 374px; border:0; transform: scale(1); overflow:hidden; margin-bottom: 3rem; margin-top: 1rem;"
```
Comment (palash23jain) [inline components/Rating/rating.html:89]: inline css can be moved to a CSS file for consistency. It is not breaking anything.

**Review by palash23jain — APPROVED** [review]
everything looks good

**Review by harshalini — APPROVED** [review]
Looks good, you can merge it👍

File: components/Rating/rating.html (hunk 2 above)
Comment (muskan-gupta-0803) [reply components/Rating/rating.html:89]: yes, will try to change it later on.

**Comment by TJain02** [comment]
The code is clean. Looks good to merge.

**Pull request merged**